.github/workflows/release-images.yml

```diff
@@ -0,0 +1,89 @@
+name: Release Images
```

Comment:
We dont release images from this repo. This would be a great action for `opencadc/canfar-library` which will build science containers, but I think this repo only validates and publishes helm chart artifacts.